crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -745,7 +745,15 @@ define_tedge_config! {
 
             /// The maximum number of software packages reported for each type of software package
             #[tedge_config(example = "1000", default(value = 1000u32))]
-            max_packages: u32
+            max_packages: u32,
+
+            /// The filtering criterion, in form of regex, that is used to filter packages list output
+            #[tedge_config(example = "^(tedge|c8y).*")]
+            include: String,
+
+            /// The filtering criterion, in form of regex, that is used to filter out packages from the output list
+            #[tedge_config(example = "^(glibc|libc|kernel-|iptables-module).*")]
```

Comment:
Sorry one minor correction (as I just tested the example on yocto):

```suggestion
            #[tedge_config(example = "^(glibc|lib|kernel-|iptables-module).*")]
```